cluster-autoscaler/core/scaledown/actuation/update_latency_tracker_test.go

```diff
@@ -196,3 +197,33 @@ func TestUpdateLatencyCalculation(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoveOldLatencies(t *testing.T) {
```

Comment:
Please use table-based tests.

Reply:
This test is removed after using expiring.List